code/modules/projectiles/projectile/special_projectiles.dm

```diff
@@ -123,6 +123,24 @@
 			shake_camera(M, 3, 1)
 	qdel(src)
 
+/obj/item/projectile/missile
+	icon = 'icons/obj/grenade.dmi'
+	icon_state = "missile"
+	damage = 50
+	var/heavy = TRUE
+
+/obj/item/projectile/missile/on_hit(atom/target, blocked, hit_zone)
+	. = ..()
```

Comment:
```suggestion
	..()
```
we don't care about the return value since you always default to `return TRUE` at the end of the proc